.circleci/config.yml

```diff
@@ -54,13 +54,22 @@ commands:
     steps:
       - restore_cache:
           keys:
-            - v1-googleko
+            - v1-googleko_bin
```

Comment:
use the ko version as part of the cache, example `v0.11.2-googleko`. Then we will know to change the cache key when we bump the ko version